Gruntfile.js

```diff
@@ -39,13 +39,21 @@ module.exports = function (grunt) {
 		nodeunit: {
 			all: 'test/node.js'
 		},
+		mochaTest: {
```

Comment:
That's weird, isn't Babel just a transpiler and Mocha just the test runner? Why would one have a dependency on the other? Can't you just run Babel standalone?